examples/wearable/UIComponents/contents/presentation-views/list/index.html

```diff
@@ -23,6 +23,11 @@ <h2 class="ui-title">
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
+				<li>
```

Comment:
to keep consistency with guide please place it after Main Text Only

Reply:
ok updated